src/styles/blocks/card.scss

```diff
@@ -0,0 +1,82 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  padding-top: 32px;
+  padding-bottom: 16px;
+  display: flex;
+  flex-direction: column;
+  flex-shrink: 0;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: $white;
+
+  &__product-img {
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__product-name {
+    margin: 40px 16px 4px;
+    color: $main-accent;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__product-code {
+    margin: 0 16px 16px;
+    min-height: 14px;
+    color: $secondary-accent;
+    font-size: 10px;
+    font-weight: 400;
+  }
+
+  &__product-price {
+    margin: 24px 16px 0;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__product-priceText {
+    color: $secondary-accent;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__product-priceValue {
+    color: $main-accent;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+  }
+
+  &__purchase-link {
+    box-sizing: border-box;
+    margin: 16px 16px 0;
+    padding: 11px 0;
+    line-height: 16px;
+    text-decoration: none;
+    font-size: 14px;
+    font-style: normal;
```

Comment:
normal is a default value for font-style

Reply:
so you can remove it
<img width="645" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/70220667/d3de5931-d9a6-42e6-b6de-086f598911e6">